src/libsync/owncloudpropagator.h

```diff
@@ -227,8 +240,20 @@ private slots:
         return next->scheduleSelfOrChild();
     }
 
+    void terminateJobs() {
+        // Send abort to the subjobs and emit abort finished signal
+        abortJobs();
```

Comment:
That's not needed, abort was already sent

So you don't need to put abortJobs in a different function